main/location.conf.template

```diff
@@ -58,6 +58,8 @@ location / {
   proxy_set_header X-Real-IP $remote_addr;
   proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
   proxy_set_header X-Forwarded-Proto $thescheme;
+  proxy_set_header X-Forwarded-Host $host;
+  proxy_set_header X-AEM-Edge-Key $AEM_EDGE_KEY
```

Comment:
Any ENV var used in nginx/lua must also be defined as an `ENV` key. This means adding `AEM_EDGE_KEY` to the [env-site.conf](https://github.com/CruGlobal/cruorg_proxy/blob/271a1d432b555380e83163cbc9907a1f33f7840c/main/env-site.conf) file.

Reply:
Docs https://nginx.org/en/docs/ngx_core_module.html#env